readme.md

```diff
@@ -1,44 +1,95 @@
-- `python generate_error_files`: running [this script](https://github.com/cfpb/hmda-test-files/blob/master/python/generate_error_files.py) produces a file that will fail conditions for each edit (other edits may also fail). These files are written to the appropriate directory based on edit type, such as /edits_files/syntax/s300.txt. The file name indicates which edit the file is designed to test. If an edit has multiple conditions, a file will be made for each condition in the format /edits_files/syntax/s301_1.txt.
+### Generating Clean Files and Edit Test Files. 
+The function method `create_files(kind)` outputs a clean test file or a set of edit test files, with an input indicating the kind of file to be created. Currently, the two inputs are `"clean_file"` which creates a clean file and `"error_files"` which produces files that will fail conditions for each edit. Edit test files are written to the directory indicated in `test_filepaths.yaml` based on edit type, such as /edits_files/test_files/Bank1/syntax/s300.txt. The file name indicates which edit the file is designed to test. If an edit has multiple conditions, a file will be made for each condition in the format /edits_files/test_files/Bank1/syntax/s301_1.txt.
 
-## Repository Structure
-- [Python](https://github.com/Kibrael/2018_test_files/tree/master/python):
-    - Contains the notebooks and python scripts used to generate synthetic LAR data, perturb the data so that it fails specific business rules, and code to check that the produced files fail as expeced.
+Clean files will pass syntax and validity edits. Edit test files will fail at least one (and sometimes more than one) edit. These files will be used as a double check for the implementation of the HMDA edits for the 2018 HMDA Platform. Creating edit test files is a two step process. The generation of edit test files requires a clean data file to be present. If edit files already exist with the same names as those produced by this script, they will be overwritten.
 
-- [Schemas](https://github.com/Kibrael/2018_test_files/tree/master/schemas)
-    - Contains JSON objects that represent the data structures for LAR and Transmittal Sheet as defined by the [2018 HMDA FIG](https://www.consumerfinance.gov/data-research/hmda/static/for-filers/2018/2018-HMDA-FIG.pdf)
+### Changing Edit Logic and the Classes Used for File Creation 
+The FileGenerator Class relies on four other classes, each with its own role in file creation. 
 
-- [Dependencies](https://github.com/Kibrael/2018_test_files/tree/master/dependancies)
-    - Contains files used in the generation of synthetic LAR data.
-        - Tract to CBSA data for 2015 (most current year)
-        - A file containing a list of US ZIP codes
+-`lar_generator` in `lar_generator.py` creates raw TS and LAR rows, the values of which are then manipulated to create clean test files or edit test files. 
```

Comment:
Can you hyperlink these files?

Reply:
Sure! These changes should be reflected in the next commit.